### DISCUSSION TITLE
Problema con identificador ${} en JavaScript

### DISCUSSION THREAD

**Discussion by a66david** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#423] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/78619937/158674078-c6bcfd6f-aba2-406b-bbc0-76e6c4e1c450.png)

Tienen alguna idea de como hacer que funcione el "${}", ya estuve buscando para solucionarlo y no encontre nada en especifico.
Gracias compañeros

**Comment by MauroMontan** [comment]
para usar ${} tiene que ser dentro de estas comillas ``.

**Comment by hackit92** [comment, marked as answer]
Esta clase de valores solo funcionan dentro de comillas invertidas, en javascript se puede utilizar comillas dobles (") simples (') e invertidas (`) según las necesidades de lo que estemos programando en esa línea de código.

**Comment by raulquintero13** [comment]
ademas estas guardando el valor en pokeInput y no lo estas utilizando

**Comment by doguedogue** [comment]
Los parámetros van dentro de comillas invertidas o backticks

En mi teclado accedo a ellas presionando:
AltGr + backtick + espacio

![IMG_20220316_165012~2.jpg](https://user-images.githubusercontent.com/23409026/158704406-bd117d18-d219-41ce-aa77-60c2d537671e.jpg)

**Comment by DanyVeneno** [comment]
a66David que tal en esta tabla puedes encontrar las combinaciones para obtener cierto tipo de caracteres https://elcodigoascii.com.ar/

**Comment by GabrielVillagran** [comment]
hola que tal, cambia  ' por comillas invertidas, son las siguientes: `
![image](https://user-images.githubusercontent.com/44887537/158747641-7af336c5-8ad2-4351-9a71-731b676605e8.png)

**Comment by Edgarsantome** [comment]
Tengo el mismo error que no se porqué no me jala

![image](https://user-images.githubusercontent.com/99237684/158892846-c42c7514-f5df-4858-abae-942a950d07d4.png)

Alguien sabe porque´??

**Reply by hackit92** [reply]
A grandes rasgos pensaría que es porqu usas el mismo nombre para función y parámetro
